scripts/package.js

```diff
@@ -1,69 +1,27 @@
-import { resolve } from 'path'
-import consola from 'consola'
 import spawn from 'cross-spawn'
-import { existsSync, readJSONSync, writeFile, copy, remove } from 'fs-extra'
-import _ from 'lodash'
+import { existsSync, remove } from 'fs-extra'
 import { rollup, watch } from 'rollup'
 import { glob as _glob } from 'glob'
 import pify from 'pify'
-import sortPackageJson from 'sort-package-json'
+
+// FIXME
+// @renoirb: I’m unsure how we should refer from here.
+//           (It's the kind of thing that made me look elsewhere than lerna)
+import { Package as BasePackage } from '../packages/normalize-package-json'
```

Comment:
@pi0 please help here :)

Reply:
Maybe if we have a fancy name like Kitty can call it KittyPackage :)